server/routes/overview/mappers/prisonerSupportNeedsMapper.ts

```diff
@@ -18,6 +18,11 @@ const toHealthAndSupportNeeds = (learnerProfile: LearnerProfile): HealthAndSuppo
       inDepthAssessmentDate: dateOrNull(learnerProfile.inDepthAssessmentDate),
       primaryLddAndHealthNeeds: learnerProfile.primaryLDDAndHealthProblem,
       additionalLddAndHealthNeeds: learnerProfile.additionalLDDAndHealthProblems?.sort() || [],
+      hasSupportNeeds:
+        !!learnerProfile.rapidAssessmentDate ||
```

Comment:
Minor, but why the double bang on rapidAssessmentDate, but not the others?

Reply:
Added the brackets